source/_patterns/00-atoms/forms/select.yaml

```diff
@@ -55,3 +62,6 @@ schema:
     - options
   dependencies:
     label: [id]
+    userInputInvalid: [message]
```

Comment:
Lines 65-67 keeps the invalid state indication separate from the messaging logic, so an informational message can be provided when the form is not invalid.